api/lib/source/polkadotV0-source.js

```diff
@@ -702,14 +703,21 @@ class polkadotAPI {
         )
         .concat(
           treasuryProposals.proposals.map(async (proposal) => {
-            // should work. treasury and council proposals are virtually the same object
+            const treasuryProposal = proposal.council[0]
+              ? proposal.council[0]
+              : undefined
```

Comment:
```suggestion

```

Reply:
? actually the change would be just leaving `proposal.council[0]` right?

Reply:
ah ok, that is what you did. GitHub wasn't displaying very clearly